src/prefect/client/client.py

```diff
@@ -1816,3 +1816,91 @@ def delete_task_run_artifact(self, task_run_artifact_id: str) -> None:
             mutation,
             variables=dict(input=dict(task_run_artifact_id=task_run_artifact_id)),
         )
+
+    def set_key_value(self, key: str, value: Any) -> str:
```

Comment:
Could you move this and your other API methods to `prefect.backend.kv_store` and have them instantiate their own Client? Otherwise we'll have to deprecate and move it as in #4439